mne/decoding/transformer.py

```diff
@@ -157,92 +159,126 @@ class EpochsVectorizer(TransformerMixin):
 
     Attributes
     ----------
-    n_channels : int
+    n_channels_ : int
         The number of channels.
-    n_times : int
-        The number of time points.
 
     """
     def __init__(self, info=None):
         self.info = info
         self.n_channels = None
```

Comment:
no need to initialize these var here.

The sklearn convention is `self.init_param = init_param`, `self.new_attribute_ = new_attribute`.